packages/tentales/src/service/__tests__/__snapshots__/utils.test.ts.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`It converts service callers 1`] = `
+Object {
+  "data": [Function],
+  "editor": [Function],
+  "renderer": [Function],
```

Comment:
Is this snapshot good enough? That each service now is a function?

Reply:
That is pretty much what the function under test does, so yeah? ;)